client/pipeline_events.go

```diff
@@ -0,0 +1,18 @@
+package client
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/calyptia/api/types"
+)
+
+// CreatePipeline within an aggregator.
```

Comment:
The code comment is wrong. I think it was just copy-pasted from `CreatePipeline`.